californium-core/src/main/java/org/eclipse/californium/core/network/config/NetworkConfig.java

```diff
@@ -69,6 +69,8 @@
 		public static final String EXCHANGE_LIFETIME = "EXCHANGE_LIFETIME";
 		public static final String NON_LIFETIME = "NON_LIFETIME";
 		public static final String MAX_TRANSMIT_WAIT = "MAX_TRANSMIT_WAIT";
+		public static final String MAX_LATENCY = "MAX_LATENCY";
+		public static final String MAX_SERVER_RESPONSE_DELAY = "MAX_SERVER_RESPONSE_DELAY";
```

Comment:
Is this the "PROCESSING_DELAY" according RFC 7252, 4.8.2, page 29?

If so, I would propose to use the term of the RFC.

Reply:
No these constants refer to RFC 7390, 2.5, page 9.
I reuse the same term and use them as suggested by sophokles73 in [https://github.com/eclipse/californium/pull/95/commits/08228a7ea7743af7ba5f91046a37b94ebd2e7e35](url)

> Instead of using the reception of a response as the trigger to finally remove the exchange from the token map, I think we should instead schedule a task for cleaning up the multicast exchange after
NON_LIFETIME + MAX_LATENCY + MAX_SERVER_RESPONSE_DELAY
as indicated by RFC 7390. This clean-up task would need to be scheduled when the original multicast request is sent out.